### PR TITLE
16.0 pre commit

### PR DESCRIPTION
**Pull request by MrGigSolutions**
Pre-commit changes to addons-container.

To review, it's easiest to go over the commits one by one. The largest commit,[[FIX] Pre-commit - Automatic fixes](https://github.com/onesteinbv/addons-container/commit/2a75c105a10f53a37f7fdb7a093e0c5bd1ce265f) consists of small changes done by pre-commit, such as changing single quotes to double quotes, and implementing better line breaks. These don't change any of the code.

The other commits do have some changes that are not very impactful, but should be checked.

### CODE REVIEW

File: multi_company_disable/static/src/js/backend.esm.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
 /** @odoo-module **/
-
-import { registry } from "@web/core/registry";
-import { } from "@web/webclient/switch_company_menu/switch_company_menu";
```
Comment (MrGigSolutions) [inline multi_company_disable/static/src/js/backend.esm.js:4]: I'm not sure why this import is here, but I removed it. If not ok, can you let me know why?

File: l10n_nl_rgs/data/account_fiscal_position_tax_template.xml [hunk 2]
```diff
@@ -55,22 +55,22 @@
         <field name="tax_dest_id" ref="btw_X3"/>
     </record>
     <!-- VAT on buying from outside the EU -->
-    <record id="position_tax_extracom_9" model="account.fiscal.position.tax.template">
+    <record id="position_tax_extracom_9_buy" model="account.fiscal.position.tax.template">
```
Comment (MrGigSolutions) [inline l10n_nl_rgs/data/account_fiscal_position_tax_template.xml:58]: These were overwritten by Odoo. Perhaps this should be checked. I have given them a unique identifier for now, and I don't see it being referenced in code anywhere.

File: account_asset_management_template/__manifest__.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # Copyright 2023 Onestein (<http://www.onestein.eu>)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
+# pylint: disable=file-not-used
```
Comment (MrGigSolutions) [inline account_asset_management_template/__manifest__.py]: Undid adding this comment in [[FIX] Pre-commit - Removed unused comment.](https://github.com/onesteinbv/addons-container/pull/109/commits/dc68f06396fbbbba39daa9d1af856c31ea7493c9).

File: multi_company_disable/static/src/js/backend.esm.js (hunk 1 above)
Comment (tarteo) [reply multi_company_disable/static/src/js/backend.esm.js:4]: Not sure either. Left over from development I guess.

File: account_configuration/__manifest__.py [hunk 4]
```diff
@@ -7,7 +7,7 @@
     "category": "Accounting & Finance",
     "license": "AGPL-3",
     "author": "Onestein BV",
-    "website": "https://www.onestein.nl",
+    "website": "https://www.onestein.eu",
```
Comment (tarteo) [inline account_configuration/__manifest__.py:10]: Actually we want .nl but it doesn't matter really.

File: l10n_nl_rgs/data/account_fiscal_position_tax_template.xml (hunk 2 above)
Comment (bvanjeelharia) [reply l10n_nl_rgs/data/account_fiscal_position_tax_template.xml:58]: @MrGigSolutions , please keep these IDs as it is. It wouldn't get overwriiten as the ids are a combination of the module name and the ID itself

**Comment by tarteo** [comment]
About the RGS mis reports and RGS module changes, the xml ids were duplicate changing. The record would be overwritten in the same file for e.g. mis_report_kpi_l10n_nl_rgs_balance_sheet_10 (line 1348, line 1422)

**Pull request merged**